### ISSUE TITLE
Be able to simulate netlists that aren't in tokenspice repo

### ISSUE THREAD

**Issue by trentmc** [tokenspice/tokenspice#48]
**Is your feature request related to a problem? Please describe.**
We recently added support for independent netlists, via #30, yay:)

Right now when a netlist file is supplied, its path is given as a relative path (e.g. `netlists/foo/netlist.py`). The `tsp` module munges the string to do a relative import of the netlist module: `import netlist.foo.netlist`.

But netlists shouldn't _have_ to live within the netlists directory. If they live there, that's fine, it means they're part of the tokenspice repo and have testing etc. But there are many use cases where the netlist lives in a directory unrelated to tokenspice, such as a different git-cloned repo. Or a SPICE-in-the-loop use case like optimization where many netlists are auto-generated over an optimization run. We should support those. That's the aim of this issue.

**Describe the solution you'd like**
Allow importing from anywhere.

It comes down to changing `tsp._importNetlistModule()`.

Here's the existing implementation.
```python
import importlib
...
def _importNetlistModule(netlist_str: str):
    module_str = netlist_str.replace('/','.').replace('.py','') 
    netlist_module = importlib.import_module(module_str)
    return netlist_module
```

This would need re-implementing such that it works for not only relative paths, but absolute paths. The Python built-in `importlib` module supports this, it's just a bit finicky.
    
The existing functionality & netlists that _are_ inside the tokenspice repo should remain. They're useful.

**Comment by trentmc** [comment]
Put to low priority; the other solidity flexibility stuff should be done first. E.g. #71, #52

**Comment by trentmc** [comment]
> the other solidity flexibility stuff should be done first. E.g. #71, #52

This is now done!

From that, brownie is set up to work with contracts in the tokenspice repo. If we want to simulate more, we copy them into `tokenspice/contracts`. It's relatively low effort to bring more contracts into that repo; the benefit of having them there is that we can write tests for them, agents around them, etc. Therefore _this_ issue is no longer important IMO. Therefore I'm closing it.
